### DISCUSSION TITLE
SSH custom list

### DISCUSSION THREAD

**Discussion by og900aero** [davatorium/rofi#1525] [General]
How can I solve that, when I switch to SSH view, I see a predefined list with the names of the servers, and if I select, connect to the selected server?

**Comment by DaveDavenport** [comment]
I don't understand the question.

**Reply by og900aero** [reply]
Now, when switch to ssh (CTRL+TAB) show the known-host list. But I want show custom list. Example: Voip, Fileserver, Icinga etc. When I select Fileserver, and press ENTER, then connect to Fileserver via ssh.

**Reply by DaveDavenport** [reply]
Add them to you ssh config file and they will show up.

```
Host fileserver
    Hostname myfileserver.net
```

**Comment by og900aero** [comment]
Whoaaa. Working great. Thank u so much :)
Of coz I use parse-known-hosts: false switch, and absolutly working great.

**Reply by DaveDavenport** [reply]
on top of that you can now do `ssh fileserver`    or `scp myfile.txt fileserver:`  too
